### ISSUE TITLE
Firebase Phone Auth SMS contains unknown App name

### ISSUE THREAD

**Issue by samitha9125** [invertase/react-native-firebase#1603]
Hi.

Since last week when I try to make a phone auth code request, the SMS I get has unknown name contained in the position of App name.

As an example, it should look like %LOGIN_CODE% is your verification code for %APP_NAME%. But what I receive is for an example,

123456 is your verification code for **Paranix**. 

I don't know what that even means since I don't have any projects or anything related to that name in firebase. Also since I'm not uploaded my app to App store or Google Play store yet, this shouldn't contain any app name at all if I'm correct.

Anyone else experiencing this? I contacted firebase support but they need to confirm this is not related to this third party library. Can someone help me to solve this?

1. Application Target Platform:
 Andriod / iOS

2. Development Operating System:

Windows 10 and macOS 10.14 (Mojave)

3. Build Tools:

Basically use VSCode for coding and XCode and Android Studio for building apps.

4. `React Native` version:
0.55.4

5. `React Native Firebase` Version:

5.0.0

6. `Firebase` Module:

Auth

7. Are you using `typescript`?
No

**Comment by Salakar** [comment]
@samitha9125 have you tried searching your project files for anything containing that name? It'll definitely be something on your end. See https://stackoverflow.com/a/52525587/2938161

I'd check your project name on the firebase console, also the project domain on the console as well - it'll be something there.

---
Loving `react-native-firebase` and the support we provide? Please consider supporting us with any of the below:

  - 👉  Back financially via [Open Collective](https://opencollective.com/react-native-firebase/donate)  
  - 👉  Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter 
  - 👉  Star this repo on GitHub ⭐️
